tests/perf-test/SelectionList.perf-test.tsx

```diff
@@ -79,14 +79,15 @@ function SelectionListWrapper({canSelectMultiple}: SelectionListWrapperProps) {
     ];
 
     const onSelectRow = (item: ListItem) => {
+        const keyForList = item.keyForList || '';
```

Comment:
I think it would be better to add:

```
if (!item.keyForList) {
  return;
}
```

Reply:
Ah yes makes sense on closer inspection of context, thanks!

Reply:
Btw, we can close this one since @mountiny already has https://github.com/Expensify/App/pull/38082